examples/shared-vpc/terraform.tfvars.example

```diff
@@ -0,0 +1,8 @@
+node_count           = "value must be either 1 or 2"
+network              = "projects/xxxxxxx/global/networks/xxxxxxx"
+subnetwork           = "xxxxxxxxx"
```

Comment:
Instead of `xxxxxxx`, can you put what a "Pre-existing subnetwork self link" looks like? A dummy value?